test/ssr/ssr.spec.ts

```diff
@@ -79,6 +79,11 @@ test.describe('SSR - Components render with shadow dom and styles', () => {
 
             // Extract the first <pie-[component]> element using a regular expression
             const pieComponentMatch = rawHtml.match(componentRegex);
+
+            if (!pieComponentMatch) {
+                console.log('Failed to find component in the SSR html: ', rawHtml)
```

Comment:
There was no visual difference in the terminal between console.log/console.error/console.warning

Reply:
I think info/warning is often allowed in linters where as log generally flags a warning. I think we still haven't added lining to this repo yet but I would suggest avoiding log